auth/user_mgt.go

```diff
@@ -665,18 +691,32 @@ func validateAndFormatMfaSettings(mfaSettings MultiFactorSettings, methodType st
 				return nil, fmt.Errorf("\"uid\" is not supported when adding second factors via \"createUser()\"")
 			}
 		case updateUserMethod:
-			if multiFactorInfo.UID == "" {
```

Comment:
Shouldn't we keep this check for `UID`?

Reply:
UID is not required while updating MFA.

Reply:
Was it required before? Are we fixing a bug here?

Reply:
No it wasn't required, it was misunderstood and overlooked previously